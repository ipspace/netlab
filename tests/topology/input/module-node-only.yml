#
# Module merge test: node-only modules
#
---
defaults:
  device: iosv

nodes:
  r1:
    module: [ bfd ]
  r2:
    module: [ bgp ]
    bgp:
      as: 65000
