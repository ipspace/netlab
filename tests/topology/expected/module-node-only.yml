bgp:
  advertise_loopback: true
  community:
    ebgp:
    - standard
    ibgp:
    - standard
    - extended
  next_hop_self: true
groups:
  as65000:
    members:
    - r2
input:
- topology/input/module-node-only.yml
- package:topology-defaults.yml
module:
- ospf
- bgp
name: input
nodes:
  r1:
    box: cisco/iosv
    device: iosv
    id: 1
    loopback:
      ipv4: 10.0.0.1/32
    mgmt:
      ifname: GigabitEthernet0/0
      ipv4: 192.168.121.101
      mac: 08-4F-A9-00-00-01
    module:
    - ospf
    name: r1
    ospf:
      area: 0.0.0.0
      router_id: 10.0.0.1
  r2:
    bgp:
      advertise_loopback: true
      as: 65000
      community:
        ebgp:
        - standard
        ibgp:
        - standard
        - extended
      neighbors: []
      next_hop_self: true
      router_id: 10.0.0.2
    box: cisco/iosv
    device: iosv
    id: 2
    loopback:
      ipv4: 10.0.0.2/32
    mgmt:
      ifname: GigabitEthernet0/0
      ipv4: 192.168.121.102
      mac: 08-4F-A9-00-00-02
    module:
    - bgp
    name: r2
ospf:
  area: 0.0.0.0
provider: libvirt
